packages/api/game/api/ws/ws.go

```diff
@@ -24,7 +24,7 @@ func HandleConnection(game *game.Game) func(http.ResponseWriter, *http.Request)
 
 		client, err := middleware.FirebaseInit(r.Context())
 		if err != nil {
-			log.Debug().Msg("FirebaseInit error server.go")
+			log.Err(err).Msgf("unexpected error")
 			http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
log is what we and http.Error what the user sees. We dont want to expose the huge stacktrace to the user, rather give him a simple message of what could have gone wrong or the "unexpected error" if unsure

Reply:
We agree on the fact that these logs are only printed in the server console which is not apparent to the user ?